my-index.ejs

```diff
@@ -11,7 +11,7 @@
             // uncomment to enable Darwin internal MSKCC service patient view link
             // uses checkDarwinAccess.do service from cbioportal/cbioportal
             // enableDarwin : true,
-            apiRoot: 'www.cbioportal.org/beta',
```

Comment:
do we want to change this?

Reply:
@alisman yeah i think so. There's a new endpoint on `/beta` that's not in production